patches/net/minecraft/server/level/ChunkMap.java.patch

```diff
@@ -8,8 +8,12 @@
              return p_140179_;
          }
      }
-@@ -535,6 +_,7 @@
+@@ -533,8 +_,11 @@
+                 this.scheduleUnload(p_140182_, p_140183_);
+             } else {
                  if (this.pendingUnloads.remove(p_140182_, p_140183_) && p_203002_ != null) {
++                    net.neoforged.neoforge.common.CommonHooks.onChunkUnload(this.poiManager, p_203002_); // Neo: PR #937
```

Comment:
I was commenting pr numbers to show what lines are patched for modders when viewing. But this line doesn’t need a comment since the hook has the whole neoforge path showing. I can take off the comment and modders can view the hook itself
 
Or was there a specific kind of comment you prefer at this spot?

Reply:
There should be documentation on why `chunkTypeCache.remove` is called, something like "Neo: prevent chunk type cache from retaining data for unloaded chunks", maybe.

Reply:
resolved now